python-sdk/src/astro/databases/google/bigquery.py

```diff
@@ -102,8 +102,9 @@ class BigqueryDatabase(BaseDatabase):
         DatabaseCustomError,
     )
 
-    def __init__(self, conn_id: str = DEFAULT_CONN_ID):
+    def __init__(self, conn_id: str = DEFAULT_CONN_ID, table: BaseTable | None = None):
         super().__init__(conn_id)
+        self.table = table
```

Comment:
@pankajastro  Don't we need a similar self.hook changes for bigquery?

Reply:
No, since bigquery hook does not expect database as kwargs